src/BasisTheory.net/Common/BaseClient.cs

```diff
@@ -166,7 +166,7 @@ private void SetRequestHeaders(HttpRequestMessage message, RequestOptions reques
         {
             var apiKey = requestOptions?.ApiKey ?? ApiKey;
             if (!string.IsNullOrEmpty(apiKey))
-                message.Headers.Add("X-API-KEY", apiKey);
+                message.Headers.Add("BT-API-KEY", apiKey);
 
             if (!string.IsNullOrEmpty(requestOptions?.CorrelationId))
                 message.Headers.Add("bt-trace-id", requestOptions.CorrelationId);
```

Comment:
We should standardize on upper or lower casing within a given sdk/client - I thought it was just mismatched in tests at first and no big deal, but we shouldn't have the SDK sending requests containing two different casings.